controller/src/main/resources/locale/locale_en_EN.properties

```diff
@@ -0,0 +1,24 @@
+id.min.error=Id: must be positive {0}
```

Comment:
why do you have both locale_en.properties and locale_en_EN.properties, 
locale_ru.properties and locale_ru_RU.properties?

Reply:
If headers not use Accept-Language, by default the server could issue an answer to both "ru" or "en" locale. In my case locale_en_EN.properties, locale_ru_Ru.properties solve this problem. Maybe is problem of my system.